src/client/datascience/interactive-window/interactiveWindowCommandListener.ts

```diff
@@ -211,21 +211,13 @@ export class InteractiveWindowCommandListener implements IDataScienceCommandList
                     // When all done, show a notice that it completed.
                     if (uri && uri.fsPath) {
                         const openQuestion1 = localize.DataScience.exportOpenQuestion1();
-                        const openQuestion2 = (await this.jupyterExecution.isSpawnSupported())
-                            ? localize.DataScience.exportOpenQuestion()
```

Comment:
Can we delete `localize.DataScience.exportOpenQuestion` now that it's no longer used in the codebase?